admin/templates/configuration/etc/qserv-czar.cnf

```diff
@@ -99,6 +99,11 @@ xrootdSpread = 0
 # This is per user query and important milestones ignore this limit.
 qMetaSecsBetweenChunkCompletionUpdates = 59
 
+# Set queryDistributionTestVer to non-zero to treat simple queries,
+# such as "SELECT COUNT(*) FROM <table>" as interactive queries to stress test
```

Comment:
And `SELECT * ` as well?

Reply:
Treating SELECT COUNT(*) and SELECT * separately would take real effort. As stated in the other comment, that configuration value should always be false outside of specific testing needs.